app/views/collection_numbers/_form.html.erb

```diff
@@ -1,9 +1,12 @@
-<% # form_tag(add_query_param(action: :new, id: @observation.id)) do %>
-<%= form_with model: :collection_number do |f| %>
+<% id = @observation&.id
```

Comment:
Something like this is needed to account for the possibility that `@observation` is `nil`. 
I don't know if it happens in real life, but it does happen in collection_numbers_controller_test.

Reply:
It seems to me like the test should supply that @observation, using `instance_variable_set`. 
If that doesn't work, please let me know, that would be good to know for me too!